Content.Server/Guardian/GuardianSystem.cs

```diff
@@ -188,6 +189,8 @@ private void UseCreator(EntityUid user, EntityUid target, EntityUid injector, Gu
             // Can only inject things with the component...
             if (!HasComp<CanHostGuardianComponent>(target))
             {
+                Loc.GetString("guardian-activator-invalid-target", ("entity", Identity.Name(target, EntityManager, user)));
+
                 _popupSystem.PopupEntity(Loc.GetString("guardian-activator-invalid-target"), user, user);
```

Comment:
Pass in the localized string we got before.
```suggestion
                _popupSystem.PopupEntity(msg, user, user);
```